src/environment.F

```diff
@@ -545,6 +546,9 @@ SUBROUTINE read_global_section(root_section, para_env, globenv)
       unit_nr = cp_print_key_unit_nr(logger, global_section, "PRINT_ELPA", extension=".Log")
       IF (unit_nr > 0) globenv%elpa_print = .TRUE.
       CALL cp_print_key_finished_output(unit_nr, logger, global_section, "PRINT_ELPA")
+#if defined(__DLAF)
```

Comment:
We try not to clutter the top level source files too much with preprocessor statements. I think, here in `environment.F` all the `#if defined(__DLAF)` could be removed.